torch/nn/utils/rnn.py

```diff
@@ -213,9 +213,9 @@ def pack_padded_sequence(
     r"""Packs a Tensor containing padded sequences of variable length.
 
     :attr:`input` can be of size ``T x B x *`` where `T` is the length of the
-    longest sequence (equal to ``lengths[0]``), ``B`` is the batch size, and
```

Comment:
> (equal to `lengths[0]`)

It wasn't true, was it?
The _maximum_ length doesn't have to be at 0.

Reply:
I suppose this was only true if ``enforce_sorted=True``

Reply:
:+1: